source/server/hot_restart_nop_impl.h

```diff
@@ -14,6 +16,10 @@ class HotRestartNopImpl : public Server::HotRestart {
 public:
   HotRestartNopImpl(){};
 
+  Thread::BasicLockable& logLock() override { return log_lock_; }
```

Comment:
nit: Move these below non-override functions, and have comment `// Server::HotRestart` above. (Easier to see overridden functions).